src/app/wizard/step/provider-settings/provider/extended/component.ts

```diff
@@ -47,13 +50,21 @@ export class ProviderExtendedComponent extends BaseFormValidator implements OnIn
   readonly Providers = NodeProvider;
   readonly Controls = Controls;
 
+  // This is only supported on AWS, Azure, GCP, and Openstack
+  supportsAllowedIPRangeOverride =
+    this.Providers.AWS || this.Providers.AZURE || this.Providers.GCP || this.Providers.OPENSTACK;
```

Comment:
How is it supposed to work? How does it check for the currently selected provider? It basically seems to be an equivalent of `true || true || true || true`.